ledger/acctonline.go

```diff
@@ -181,7 +181,7 @@ func (ao *onlineAccounts) initializeFromDisk(l ledgerForTracker, lastBalancesRou
 		return
 	}
 
-	ao.accountsq, err = ao.dbs.MakeOnlineAccountsReader()
+	ao.accountsq, err = ao.dbs.MakeOnlineAccountsOptimizedReader()
```

Comment:
This doesn't look like a move/rename based refactor, was this intended?

Looking through the other changed files now to find matching changes but I was only seeing `store` -> `trackerdb`

Reply:
Yea, it's just a name change to match the pattern we are using (you are probably thinking it does something different based on the `TransactionScope` methods). 
The dbs always returned the optimized but the method was added before we had the 2 types of clients, so it never had the "optimized" in it.